examples/component/package.json

```diff
@@ -1,13 +1,25 @@
 {
-  "name": "@example/component",
-  "version": "0.0.1",
-  "private": true,
-  "scripts": {
-    "start": "astro --root demo dev",
-    "build": "astro --root demo build",
-    "serve": "astro --root demo preview"
+  "name": "my-component-library",
+  "version": "0.1.0",
+  "type": "module",
+  "exports": {
+    ".": "./index.ts"
   },
-  "dependencies": {
-    "astro": "^1.1.5"
+  "files": [
+    "src",
+    "index.ts"
+  ],
+  "keywords": [
+    "astro-component"
+  ],
+  "scripts": {},
+  "devDependencies": {
+    "@types/node": "^18.6.2",
+    "@types/prettier": "^2.6.4",
+    "astro": "^1.1.2",
+    "typescript": "^4.7.4"
```

Comment:
Doesn't really need the two types packages and typescript. They're good additions but we don't have them in the other packages